app/queues/certificate/certificate-error-handler.server.ts

```diff
@@ -57,6 +57,12 @@ export const initCertificateErrorHandler = () => {
 
       await certificateModel.updateCertificateById(certificateId, {
         status: CertificateStatus.failed,
+        /**
+         * Setting orderUrl to null, so if we get back the same cert request from the
+         * ACME provider next time (as it deduplicates orders) we are not violating the
+         * unique constraint of this field
```

Comment:
What is the value of the unique constraint, since we update the certificate by it's `id`?

Reply:
Yes, I understand why this is happening.  What I want to know is, why are we saying this URL has to be unique?  We've just seen that in normal circumstances, it can be duplicated with a repeat attempt on a failed cert order.  What was the reason that we made this field `@unique`, and should we remove that?

Reply:
The unique there ensures that a single user cannot run two cert requests in parallel (they'd get back the same order url twice, whouch would force one of the requests to fail just like it did now). Plus I didn't want to do a migration to remove it :)